ui/styl/app.styl

```diff
@@ -58,3 +58,25 @@ body
   text-transform: none
   vertical-align: -2px
 
+.stats-table
```

Comment:
I would say not yet.  This can be broken up easily in one code review at any time.